### PR TITLE
Name 'amount' the Balance field of Transfer txs

### LINKED ISSUE #591: Make transfer message fields consistent
Reported by the upstream, one of the transfer tx message field for the balance is `amount` while the other one is `value`.

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Closes #591

### CODE REVIEW

File: core/src/message.rs [hunk 1]
```diff
@@ -213,33 +213,33 @@ pub struct SetCheckpoint {
...
 ///
-/// The org account must have a balance of at least `value`.
+/// The org account must have a balance of at least `amount`.
 ///
 #[derive(Decode, Encode, Clone, Debug, Eq, PartialEq)]
 pub struct TransferFromOrg {
     pub org_id: Id,
     pub recipient: AccountId,
-    pub value: Balance,
+    pub amount: Balance,
```
Comment (CodeSandwich) [inline core/src/message.rs:222]: Shouldn't `Balance` be renamed then too?

Comment (NunoAlexandre) [reply core/src/message.rs:222]: Don't think so. `Balance` is the type, `amount` is the field. `Amount` is "the amount being transferred", `Balance` represents a an amount in a specific currency. If you would like to discuss that further feel free to follow it up on a new issue, here I am just really concerned with making these fields consistent between themselves and with the CLI :)

**Comment by NunoAlexandre** [comment]
@CodeSandwich, I have bumped the `impl_version` and ran `./scripts/rebuild-chain-specs impl`, pushed in https://github.com/radicle-dev/radicle-registry/pull/592/commits/052aec415890215fa928881e56e4bed742e31773.

Will wait for your signal to merge.

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@ edition = "2018"
 name = "radicle-registry-runtime"
 description = "Substrate chain runtime for the Radicle Registry"
 authors = ["Monadic GmbH <radicle@monadic.xyz>"]
-version = "0.16.0"
+version = "1.16.0"
```
Comment (CodeSandwich) [inline runtime/Cargo.toml:6]: Wrong way :scream: 
```suggestion
version = "0.16.1"
```

Comment (NunoAlexandre) [reply runtime/Cargo.toml]: :man_facepalming: what was I thinking about

File: Cargo.lock [hunk 3]
```diff
@@ -3805,7 +3805,7 @@ dependencies = [
 
 [[package]]
 name = "radicle-registry-runtime"
-version = "0.16.0"
+version = "1.16.0"
```
Comment (CodeSandwich) [inline Cargo.lock:3808]: The lock needs a fix too. I think that it's the last place.
```suggestion
version = "0.16.1"
```

**Comment by NunoAlexandre** [comment]
Thanks for the patience, @CodeSandwich!

**Pull request merged**